packages/scene-composer/src/hooks/useMatterportTags.ts

```diff
@@ -27,6 +50,9 @@ const addTag = (
       matterportId: id, //mattertag uses item.sid and tag uses item.id so we just us the collection key for both
     },
   } as ISceneNode;
+
+  const dataoverlayComponent = getNewDataOverlayComponent(item);
+  node.components?.push(dataoverlayComponent);
```

Comment:
can we create the overlay before creating the node and directly adding to the components at line43? having this part separately will easily get missed

Reply:
Done